### PR TITLE
Improve tests

### PR DESCRIPTION
**Pull request by jwierzbo**

### CODE REVIEW

File: client_server_info_impl.go [hunk 1]
```diff
@@ -140,6 +140,13 @@ func (c *client) echo(ctx context.Context) error {
 	if err != nil {
 		return WithStack(err)
 	}
+
+	// Velocypack requires non-empty body for versions < 3.11.
+	req, err = req.SetBody("echo")
```
Comment (nikita-vanyasin) [inline client_server_info_impl.go:145]: Is this "echo" value somehow a specific keyword? Maybe it worth moving into constant and re-use that?

**Review by informalict — APPROVED** [review]
LGTM

File: client_server_info_impl.go (hunk 1 above)
Comment (informalict) [reply client_server_info_impl.go:145]: no, `echo` is just an any body. It can be any non empty string

Comment (jwierzbo) [reply client_server_info_impl.go:145]: @nikita-vanyasin This `echo` method is like `echo` in bash - it returns the same request as you send

**Pull request merged**